executor/adapter.go

```diff
@@ -611,6 +611,16 @@ func (a *ExecStmt) handleForeignKeyTrigger(ctx context.Context, e Executor, dept
 	return nil
 }
 
+// handleForeignKeyCascade uses to execute foreign key cascade behaviour, the progress is:
+//  1. Build delete/update executor for foreign key on delete/update behaviour.
+//     a. Construct delete/update AST. We used to try generated SQL string first and then parse the SQL to get AST,
+//     but we need convert Datum to string, there may be some risks here, so we chose to construct AST directly.
```

Comment:
any risk explanation comment here？

Reply:
The parse and restore function may not be a bijection.

```
Datum a = ...;
assert_eq(parse(restore(a))); -- this assertion may be broken
```

Reply:
great, already add this into comment.